filebeat/_meta/fields.common.yml

```diff
@@ -23,6 +23,12 @@
       description: >
         The content of the line read from the log file.
 
+    - name: stream
```

Comment:
Could we call this `log.stream` or something similar? `log.source`? Source is already overloaded.

@weltenwort has some ideas for a better name?